services/sts/systests/advanced/src/test/java/org/apache/cxf/systest/sts/kerberos/KerberosDelegationTokenTest.java

```diff
@@ -135,6 +138,48 @@ public void testKerberosTokenJAXRS() throws Exception {
         org.junit.Assert.assertEquals(2 * numToDouble, resp);
     }
 
+    @org.junit.Test
+    public void testKerberosTokenJAXRSCustomLogin() throws Exception {
```

Comment:
This test class is not actually run (there is a junit Ignore specified on the class). Could you instead move the test to the systests/kerberos directory? 

https://github.com/apache/cxf/tree/master/systests/kerberos/src/test/java/org/apache/cxf/systest/kerberos/jaxrs/kerberos

The test should show the need for this PR by failing before the fix was applied, and passing after.